lib/sentry/plug_context.ex

```diff
@@ -152,7 +175,7 @@ defmodule Sentry.PlugContext do
       |> Plug.Conn.fetch_query_params()
 
     %{
-      url: Plug.Conn.request_url(conn),
+      url: apply_fun_with_conn(conn, url_scrubber),
```

Comment:
Using `apply_fun_with_conn` here is consistent with how the other scrubbers are called.

However, it also has the effect that passing in `nil` for `url_scrubber` will cause `%{}` to be sent instead of `Plug.Conn.request_url(conn)`.

Does this matter? Should `apply_fun_with_conn` get a new 3rd `default` argument that defaults to `%{}`?

```elixir
defp apply_fun_with_conn(conn, function, default \\ %{})
defp apply_fun_with_conn(_conn, _function = nil, default), do: default
defp apply_fun_with_conn(conn, {module, fun}, _default), do: apply(module, fun, [conn])
defp apply_fun_with_conn(conn, fun, _default) when is_function(fun, 1), do: fun.(conn)
```

```elixir
url: apply_fun_with_conn(conn, url_scrubber, Plug.Conn.request_url(conn)),
```

Reply:
Yes let's go with a third argument, but without the `\\` clause. Just do `apply_fun_with_conn(conn, function, default)` and explicitly pass in `%{}` in the other instances where we call `apply_fun_with_conn/2` today.

Reply:
Will do! I like the explicitness.